### ISSUE TITLE
ModuleNotFoundError: No module named 'sklearn.externals.joblib' with sklearn 0.23.1

### ISSUE THREAD

**Issue by ricmarchao** [petersontylerd/mlmachine#6]
hi

I was reading :
https://towardsdatascience.com/mlmachine-groupbyimputer-kfoldencoder-and-skew-correction-357f202d2212

decided to install the package

but got an error since I am on sklearn 0.23.1

  File "D:\Anaconda3\envs\py37\lib\site-packages\mlmachine\features\preprocessing.py", line 41, in <module>
    from sklearn.externals.joblib import Parallel, delayed

ModuleNotFoundError: No module named 'sklearn.externals.joblib'

I think with sklearn 0.23.1 joblib is imported directly from joblib

a fix could be:
from joblib import Parallel, delayed

**Comment by rick-62** [comment]
I got the same error. 

I reverted scikit-learn version to 0.22.2.post1, which is listed as the minimum requirement. 

When I import mlmachine, I get the following warning:
```
FutureWarning: sklearn.externals.joblib is deprecated in 0.21 and will be removed in 0.23. 
Please import this functionality directly from joblib, which can be installed with: pip install joblib. 
If this warning is raised when loading pickled models, you may need to re-serialize those models with scikit-learn 0.21+.
```

**Comment by petersontylerd** [comment]
Thank you for checking out mlmachine and for bringing this to my attention. I'll take care of it.

**Comment by petersontylerd** [comment]
Fixed with mlmachine version 0.1.5

pip install -U mlmachine==0.1.5

**Comment by alejandrohdo** [comment]
The solution is to install: `pip install scikit-learn==0.22.2.post1`
